goldmane/cmd/health/main.go

```diff
@@ -0,0 +1,58 @@
+package main
```

Comment:
Would this be a candidate for something more central that all our apis could use?

Reply:
Yeah, I think it could be